### ISSUE TITLE
please add prerequisites for package version

### ISSUE THREAD

**Issue by hitercs** [xwhan/DeepPath#8]
Hi,

Could you please add the prerequisites package (eg. tensorflow version) for this repo.?
Thanks.

**Comment by hitercs** [comment]
along with the keras version

**Comment by ouening** [comment]
I have successfully test the code in colab, and I found that the enviroment is:
python: 2.7
tensorflow: 1.13.1 (can be installed in colab using commands below:
```
!python2 -m pip uninstall tensorflow
!python2 -m pip install tensorflow-gpu==1.13.1
```
 At last I found out that there is no `transH_eval.py` and `transD_eval.py` file in repository.
My colab link is : https://colab.research.google.com/drive/1sWkwUwOWvWDpkIPLEQ1YXz5SvZ0zfzbo
Note: You should change your own working directory in your Google Drive!

**Comment by chrislouis0106** [comment]
Thank you, it's very helpful. As you said, the TranH and TranD isn't available. So if you find the codes about the model.

**Comment by Nikronic** [comment]
@ouening I found the error can be solved using this issue regarding `evaluation.py`  https://github.com/xwhan/DeepPath/issues/9

**Comment by elizeBu** [comment]
> I have successfully test the code in colab, and I found that the enviroment is:
> python: 2.7
> tensorflow: 1.13.1 (can be installed in colab using commands below:
> 
> ```
> !python2 -m pip uninstall tensorflow
> !python2 -m pip install tensorflow-gpu==1.13.1
> ```
> 
> At last I found out that there is no `transH_eval.py` and `transD_eval.py` file in repository.
> My colab link is : https://colab.research.google.com/drive/1sWkwUwOWvWDpkIPLEQ1YXz5SvZ0zfzbo
> Note: You should change your own working directory in your Google Drive!

Yes, I agree with you. Did you run FB15k-237 dataset in your colab? I do not know if you successfully get the result. I found some issues occured in this dataset called "A.bern". Do you know what this file exactly is and what it used for?

**Comment by XJCM1** [comment]
> I have successfully test the code in colab, and I found that the enviroment is: python: 2.7 tensorflow: 1.13.1 (can be installed in colab using commands below:
> 
> ```
> !python2 -m pip uninstall tensorflow
> !python2 -m pip install tensorflow-gpu==1.13.1
> ```
> 
> At last I found out that there is no `transH_eval.py` and `transD_eval.py` file in repository. My colab link is : https://colab.research.google.com/drive/1sWkwUwOWvWDpkIPLEQ1YXz5SvZ0zfzbo Note: You should change your own working directory in your Google Drive!

hello，if i dont use colab，can i run it with my own computer?why it cannot run with python2.7 and tf1.13.1,hope for your reply.
